include/deal.II/grid/grid_tools_cache.h

```diff
@@ -150,6 +150,26 @@ namespace GridTools
     const Mapping<dim, spacedim> &
     get_mapping() const;
 
+
+    /**
+     * Its elements are pairs of bounding boxes denoting areas
```

Comment:
That's not a good start to a paragraph. You need to start with a sentence that explains what the "Its" here refers to. Maybe something like "This function returns an object that allows identifying which process(es) in a parallel computation may own the cell that surrounds a given point. The elements of this object -- an Rtree -- are pairs..."